<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>c_recipeView</class>
 <widget class="QWidget" name="c_recipeView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>941</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="milestonesScroll">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>132</width>
     <height>99</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="milestoneArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>116</width>
      <height>97</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="titleRecipeEdit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="titleRecipe">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIngredients">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingrédients :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEquipment">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Matériel :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelProcesses">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>400</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Temps :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
